pkg/network/tracer/connection/tracer.go

```diff
@@ -525,12 +526,12 @@ func populateConnStats(stats *network.ConnectionStats, t *netebpf.ConnTuple, s *
 		Cookie:           s.Cookie,
 	}
 
+	stats.ProtocolStack = protocols.NewStack(
+		s.Protocol_stack.Api,
+		s.Protocol_stack.Application,
+		s.Protocol_stack.Encryption,
```

Comment:
Are these always present?

Reply:
Not necessarily (I mean, they can be set to zero if that's what you're asking)